### PR TITLE
fix(LT-5967): update expiration header conversions

### PR DESCRIPTION
**Pull request by tarurar**

### CODE REVIEW

File: src/Lykke.RabbitMqBroker/ArgumentsBuilderExtensions.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+using Lykke.RabbitMqBroker.Subscriber;
+
+namespace Lykke.RabbitMqBroker;
+
+internal static class ArgumentsBuilderExtensions
+{
+    public static uint ToExpirationMilliseconds(this double src) => src switch
+    {
+        > uint.MaxValue => throw new ArgumentOutOfRangeException(nameof(src), "Value is too large to be converted to uint"),
```
Comment (kobzev) [inline src/Lykke.RabbitMqBroker/ArgumentsBuilderExtensions.cs:11]: it's 50 days
maybe ulong is better?

image
image

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+## [[tbd]] ([[date]])
+
+### Fixed
+- LT-5967: Fix `x-expires` argument header conversions
```
Comment (kobzev) [inline CHANGELOG.md:4]: overflow header was touched too

File: src/Lykke.RabbitMqBroker/Subscriber/MessageReadStrategies/QueueDeclarationArgumentsBuilder.cs [hunk 3]
```diff
@@ -58,14 +60,14 @@ public DealLetterExchangeArgumentsBuilder(QueueDeclarationArgumentsBuilder paren
     public DealLetterExchangeArgumentsBuilder WithAtLeastOnceStrategy()
     {
         _arguments["x-dead-letter-strategy"] = "at-least-once";
-        _arguments["overflow"] = "reject-publish";
+        _arguments[Headers.XOverflow] = "reject-publish";
```
Comment (kobzev) [inline src/Lykke.RabbitMqBroker/Subscriber/MessageReadStrategies/QueueDeclarationArgumentsBuilder.cs:63]: is this an equivalent change?

Comment (tarurar) [reply src/Lykke.RabbitMqBroker/Subscriber/MessageReadStrategies/QueueDeclarationArgumentsBuilder.cs:63]: It was a bug before. Right header name is `x-overflow`.

File: CHANGELOG.md (hunk 2 above)
Comment (tarurar) [reply CHANGELOG.md:4]: Fixed

File: src/Lykke.RabbitMqBroker/ArgumentsBuilderExtensions.cs (hunk 1 above)
Comment (tarurar) [reply src/Lykke.RabbitMqBroker/ArgumentsBuilderExtensions.cs:11]: I'll try to use ulong instead.

**Review by kobzev — APPROVED** [review]

**Pull request merged**